docs/git-workflow.md

```diff
@@ -1,36 +1,67 @@
 # Git branching workflow for narrative
 
-- Based on: http://drewfradette.ca/a-simpler-successful-git-branching-model/
-- Last modified: Dec. 11, 2014
-
 This document describes the git workflow that *all* developers of the narrative code must follow. It also describes the release procedure, which only an unlucky few should have to worry about.
 
+- Last modified: Aug. 31, 2020
+
+***Table of Contents***
+
+- [Branches and Tags](#branches-tags)
+- [Contributing](#Contributing)
+  - [Pull Requests](#pull-requests)
+- [Release Flows](#realease-flow)
+  - [Development Releases](#development-releases)
+  - [Production Releases](#production-releases)
+  - [Hotfix Releases](#hotfixes-releases)
+
 ## Branches and Tags
 
 There are 3 branches:
 
-* `develop` – Development branch. New features are added off this branch.
-* `staging` – Staging or "release candidate" branch. The branch deployed `narrative-dev`.
-* `master` – Production branch. The branch deployed on `narrative-next` and `narrative`. This branch is used for hotfixes.
+- `develop` – Development branch. New features are added off this branch.
+- `staging` – Staging or "release candidate" branch. The branch deployed `narrative-dev`.
```

Comment:
This particular line is pretty outdated. This afternoon's conversation aside, we really only have `develop` and `master` as the primary branches. I think you can safely delete that line. I'm sure there'll be some follow-up later to discuss feature branches and their deployment.